docket.spec

```diff
@@ -1,7 +1,7 @@
 %global _docketdir /opt/rocknsm/docket
 
 Name:           docket
-Version:        0.1.1
+Version:        0.2.1
```

Comment:
Drop this back to `0.1.1`. I manage the version number with [tito](https://github.com/dgoodwin/tito). After this merges, I'll maybe bump it to `1.0.0`.

Reply:
Disregard that. I see your commit logs below. Leave this. I'll still bump it after merge.